scripts/run-dependencies.sh

```diff
@@ -40,6 +35,41 @@ $here/montagu_cli.sh add "Test User" test.user \
 
 $here/montagu_cli.sh addRole test.user user
 
-# Add user to orderlyweb
-$here/orderlyweb_cli.sh add-users test.user@example.com
-$here/orderlyweb_cli.sh grant test.user@example.com */reports.read */reports.run */reports.review
+# Run packit
+hatch env run pip3 install constellation
+hatch env run pip3 install packit-deploy
```

Comment:
if we're using hatch, why not add these to the pyproject.toml so they're installed when hatch builds the env?

Reply:
I don't actually see how this is working - I'll follow up in person